compiler/rustc_llvm/llvm-wrapper/RustWrapper.cpp

```diff
@@ -334,6 +334,16 @@ extern "C" void LLVMRustAddStructRetAttr(LLVMValueRef Fn, unsigned Index,
   AddAttribute(F, Index, Attr);
 }
 
+extern "C" void LLVMRustEmitUWTableAttr(LLVMValueRef Fn, bool Async) {
+  Function *F = unwrap<Function>(Fn);
+#if LLVM_VERSION_LT(15, 0)
+  Attribute Attr = Attribute::get(F->getContext(), Attribute::UWTable);
+#else
+  Attribute Attr = Attribute::getWithUWTableKind(F->getContext(), Async ? UWTableKind::Async : UWTableKind::Sync);
```

Comment:
tidy is sad about the long line :)

Reply:
Yep, got that and the trailing ws issue too. I guess rustfmt doesn't fix trailing ws in comments. :(